src/Container.php

```diff
@@ -36,10 +41,8 @@ class Container implements ContainerInterface, ArrayAccess
      * @param array|ArrayAccess                             $config
      * @param \League\Container\Definition\FactoryInterface $factory
```

Comment:
`null` is missing from here: `@param \League\Container\Definition\FactoryInterface|null $factory`